### PR TITLE
Qt: Show disc number

### PR DESCRIPTION
**Pull request by spycrab**

### CODE REVIEW

**Comment by spycrab** [comment]
@JosJuice Is this reliable though?

**Comment by JosJuice** [comment]
I don't know if it covers every single game in every single language, but it seems to have worked out well enough in DolphinWX.

**Comment by BhaaLseN** [comment]
Some braces would be nice too, but otherwise it seems to do just fine:
image

File: Source/Core/DolphinQt2/GameList/GameListModel.cpp [hunk 1]
```diff
@@ -74,7 +74,19 @@ QVariant GameListModel::data(const QModelIndex& index, int role) const
...
     if (role == Qt::DisplayRole || role == Qt::InitialSortOrderRole)
-      return QString::fromStdString(game.GetName());
+    {
+      QString name = QString::fromStdString(game.GetName());
+      const int disc_nr = game.GetDiscNumber() + 1;
+      if (game.GetDiscNumber() != 0)
+      {
+        if (!name.contains(QRegExp(QStringLiteral("disc ?%1").arg(disc_nr), Qt::CaseInsensitive)))
+        {
+          name.append(game.GetDiscNumber() != 0 ? tr(" (Disc %1)").arg(disc_nr) :
```
Comment (JosJuice) [inline Source/Core/DolphinQt2/GameList/GameListModel.cpp]: You've already checked `game.GetDiscNumber() != 0` four lines above, so you don't need a ternary operator here.

File: Source/Core/DolphinQt2/GameList/GameListModel.cpp [hunk 2]
```diff
@@ -74,7 +74,18 @@ QVariant GameListModel::data(const QModelIndex& index, int role) const
     break;
   case COL_TITLE:
     if (role == Qt::DisplayRole || role == Qt::InitialSortOrderRole)
-      return QString::fromStdString(game.GetName());
+    {
+      QString name = QString::fromStdString(game.GetName());
+      const int disc_nr = game.GetDiscNumber() + 1;
+      if (game.GetDiscNumber() != 0)
```
Comment (BhaaLseN) [inline Source/Core/DolphinQt2/GameList/GameListModel.cpp]: Thoughts on `disc_nr > 1`?

**Pull request merged**